assets/js/experience.js

```diff
@@ -1,3 +1,62 @@
+//  Work experience cards
+
+
+const experiencecards = document.querySelector(".experience-cards");
+const exp = [
+  {
+    title: "Reasearch and Development Intern",
+    cardImage: "assets/images/experience-page/UCL.JPG",
```

Comment:
@Adyasha8105 Nice work. Just replace this with some other data so that this repository is more generalized. Have a look at Deedy resume and do the changes accordingly please